test/toolkits/test_openai_function.py

```diff
@@ -88,7 +88,7 @@ def test_all_parameters(
                     },
                     'default_enum_para': {
                         'default': 'critic',
-                        'allOf': [{'$ref': '#/definitions/RoleType'}],
+                        '$ref': '#/definitions/RoleType',
```

Comment:
Thanks @WHALEEYE , I’m curious about the rationale behind Pydantic’s removal of `all0f` and whether this change will impact our tool-calling implementation. According to https://github.com/pydantic/pydantic/issues/478 , `all0f` appears to be useful in certain scenarios, I hope we can gain some insights beyond simply modifying our test code to pass the tests.